src/main/java/edu/kis/powp/jobs2d/visitor/DriverCommandVisitor.java

```diff
@@ -3,18 +3,16 @@
 import java.util.ArrayList;
 import java.util.List;
 
-import edu.kis.powp.jobs2d.Job2dDriver;
-
-public class DriverCommandVisitor implements Visitable {
-    private List<Visitable> elements = new ArrayList<>();
+public class DriverCommandVisitor implements VisitableDriverCommand {
```

Comment:
This is not a visitor class! 
Actually I do not see a point for introducing it. You should apply visitor ICompoundCommand.